packages/mocaccino-dracut/mocaccino-dracut

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash
+# Author: geaaru@sabayonlinux.org
```

Comment:
What about also adding you here now since you refactored it ? :)

Reply:
LOL, I appreciate the thought. I still think geaaru@ is the author. I'm more like the editor LOL.